docs/rfc/001-block-retention.md

```diff
@@ -0,0 +1,108 @@
+# RFC 001: Configurable Block Retention
```

Comment:
We explicitly removed the docs/rfc and docs/architecture directories from the backport branches, to avoid confusion (see https://github.com/tendermint/tendermint/issues/7865)

Reply:
Right that makes sense